src/Sfa.Tl.ResultsAndCertification.Web/Views/AdminDashboard/AdminReviewChangesIndustryPlacement.cshtml

```diff
@@ -0,0 +1,114 @@
+@model AdminReviewChangesIndustryPlacementViewModel
+@using Sfa.Tl.ResultsAndCertification.Web.ViewModel.AdminDashboard.IndustryPlacement
+@using Sfa.Tl.ResultsAndCertification.Common.Helpers
+@using Sfa.Tl.ResultsAndCertification.Web.Content.AdminDashboard
+
+@{
+    ViewData["Title"] = ReviewChangesIndustryPlacement.Page_Title;
+}
+
+@section NavigationBar
+{
+    <vc:back-link model=Model.BackLink></vc:back-link>
+}
+
+<div class="govuk-grid-row">
+    <div class="record-header">
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-0">
+            @ReviewChangesIndustryPlacement.Heading_Change_Start_Year
+        </h1>
+    </div>
+    <partial name="_ValidationSummary" />
```

Comment:
I think all this should be a partial view. It is going to look the same in all the review page journeys. Same with the page bottom. Contact name, Zendesk ID, etc. All pages are going to be the same except the centre part.